src/schema/examples/msIdentifier.xml

```diff
@@ -2,6 +2,10 @@
 <egXML xmlns="http://www.tei-c.org/ns/Examples">
   <msIdentifier>
     <idno source="https://suche.staatsarchiv.djiktzh.ch/detail.aspx?ID=3737536" xml:lang="de">StAZH A III 2, S. 353-355</idno>
-    <repository xml:lang="de">Staatsarchiv Zürich</repository>
+    <repository xml:lang="de">Staatsarchiv</repository>
```

Comment:
Das ist so nicht korrekt. Das Staatsarchiv des Kantons Zürich wird von uns immer unter dem Namen Staatsarchiv Zürich geführt. Wir sollten das auch bei den Beispielen beibehalten.

Reply:
Beispiel in settlement.xml falsch, Änderung umgesetzt.